6_0_upgrade.md

```diff
@@ -6,20 +6,20 @@ straightforward.
 
 ## Preparation
 
-- If your `source_path` is `app/javascript`, rename it to `app/packs`
-- If your `source_entry_path` is `app/javascript/packs`, rename it to `app/packs/entrypoints`
-- Rename `config/webpack` to `config/webpack_old`
-- Rename `config/webpacker.yml` to `config/webpacker_old.yml`
-- Uninstall the current version of `webpack-dev-server`: `yarn remove webpack-dev-server`
-- Upgrade webpacker
+1. If your `source_path` is `app/javascript`, rename it to `app/packs`
+2. If your `source_entry_path` is `app/javascript/packs`, rename it to `app/packs/entrypoints`
```

Comment:
Using 1.0 for all options is a useful trick to make re-ordering/adding easier in future